src/components/Datepicker.vue

```diff
@@ -9,6 +9,11 @@
     @keydown.esc="resetOrClose"
     @keydown.tab="tabThroughNavigation($event)"
   >
+    <p>
+      <b>selectedDate:</b>
+      {{ selectedDate }}
+    </p>
```

Comment:
Debut output?

Reply:
Yes - it was just to show you how it works. I've dropped that commit.